.pycodestyle.ini

```diff
@@ -5,4 +5,5 @@
 # E501: line too long (checked by PyLint)
 # E711: comparison to None (used to improve test style)
 # E712: comparison to True (used to improve test style)
-ignore = E401,E402,E501,E711,E712
+# E741 ambiguous variable name
```

Comment:
I think that ultimately we'll want to bring this check (E741) back. It was complaining about the use of `l` as a variable name.